src/index.d.ts

```diff
@@ -1204,43 +1204,23 @@ declare namespace admin.securityRules {
 
 declare namespace admin.machineLearning {
   /**
-   * Interface representing options for listing Models.
+   * Firebase ML Model input objects
    */
-  interface ListModelsOptions {
-    /**
-     * An expression that specifies how to filter the results.
-     *
-     * Examples:
-     *
-     * ```
-     * display_name = your_model
-     * display_name : experimental_*
-     * tags: face_detector AND tags: experimental
-     * state.published = true
-     * ```
-     *
-     * See https://firebase.google.com/docs/ml-kit/manage-hosted-models#list_your_projects_models
-     */
-    filter?: string;
-
-    /** The number of results to return in each page. */
-    pageSize?: number;
-
-    /** A token that specifies the result page to return. */
-    pageToken?: string;
+  interface ModelOptionsBase {
+    displayName?: string;
+    tags?: string[];
   }
-
-  /** Response object for a listModels operation. */
-  interface ListModelsResult {
-    /** A list of models in your project. */
-    readonly models: Model[];
-
-    /**
-     * A token you can use to retrieve the next page of results. If null, the
-     * current page is the final page.
-     */
-    readonly pageToken?: string;
+  interface GcsTfliteModelOptions extends ModelOptionsBase {
+    tfliteModel: {
+      gcsTfliteUri: string;
+    };
   }
+  interface AutoMLTfliteModelOptions extends ModelOptionsBase {
+    tfliteModel: {
+      automlModel: string;
+    };
+  }
+  type ModelOptions = ModelOptionsBase | GcsTfliteModelOptions | AutoMLTfliteModelOptions;
```

Comment:
Should ModelOptionsBase be included in this union?

Reply:
yes